mesonbuild/interpreter/interpreter.py

```diff
@@ -3338,6 +3338,21 @@ def build_target_decorator_caller(self, node, args, kwargs):
             if kwargs['win_subsystem'] is None:
                 kwargs['win_subsystem'] = 'console'
 
+            if kwargs['implib']:
+                if kwargs['export_dynamic'] is False:
+                    FeatureDeprecated.single_use('implib overrides explict export_dynamic off', '1.3.0', self.subprojct,
+                                                 'Do not set ths if want export_dynamic disabled if implib is enabled',
```

Comment:
I would offer a spelling correction but I'm not completely sure what the message is trying to say.